helm/vmss-prototype/templates/vmss-prototype.yaml

```diff
@@ -10,6 +10,35 @@
 {{- end -}}
 {{- end -}}
 
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kamino
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: node-updater
+rules:
+- apiGroups: [""]
+  resources: ["nodes"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: update-nodes
```

Comment:
As above - the names should likely be derived from the job name as the below shows.  The reason being that we don't want two different deployments of this chart into the same cluster to step on each other.